src/apps/components/AppInstallPage/AppInstallPage.tsx

```diff
@@ -39,6 +41,18 @@ export const AppInstallPage: React.FC<AppInstallPageProps> = ({
 }) => {
   const intl = useIntl();
   const classes = useStyles();
+  const { theme } = useTheme();
+
+  const getSaleorLogoUrl = () => {
```

Comment:
We already have a themeable Saleor logo in the side navigation. Is the code duplicated, or was there another reason?

Reply:
Previously it was using a fixed image. In this diff I replaced it to one that is used in the navigation. Its not themable - these are 2 different PNGs